src/Kopernicus/Configuration/PropertiesLoader.cs

```diff
@@ -338,6 +337,10 @@ void IParserEventSubscriber.Apply(ConfigNode node)
             // PostApply Event
             void IParserEventSubscriber.PostApply(ConfigNode node)
             {
+                // Replace biomes
+                if (biomes?.Count() > 0)
+                    celestialBody.BiomeMap.Attributes = biomes.Select(b => b.attribute).ToArray();
```

Comment:
Can you add a check if the biome map is null, and if yes, throw an appropriate exception?